community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java

```diff
@@ -57,7 +57,6 @@
 import org.neo4j.unsafe.impl.batchimport.store.io.IoMonitor;
 
 import static java.lang.System.currentTimeMillis;
-
```

Comment:
I reformatted with the neo4j scheme that I've got installed and it still seems to remove that line. Why should there be a blank line after that import?